drv/sidecar-seq-server/src/clock_generator_payload_b.rs

```diff
@@ -782,7 +782,7 @@ pub fn idt8a3xxxx_payload<E>(
             0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
             0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
             0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-            0x00, 0x00, 0x00, 0x01, 0x00, 0x00,
+            0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
```

Comment:
Is it expected that this value changed, but all of the comments above stayed the same?

Reply:
Yes. This change indicates that particular DPLL does not track input[1] anymore. But the parser used to generate this does not seem to pick that up.